packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -80,6 +80,7 @@ export function generateRuntimeOptions(
         useDataStoreAliasing: [undefined],
         enableOfflineLoad: [undefined],
         flushMode: [undefined],
+        compressionOptions: [undefined],
```

Comment:
You can actually enable compression here - this is used only in our stress tests, so that's the way to enable it while it's disabled in shipping code. That way we can start getting data, including perf data BTW!

Reply:
We might have no large ops in stress tests though. So maybe some tuning on cut-off is required specifically for stress test - you can run them locally on your machine and experiment.

Reply:
From local testing, it seems like 500 will work as a cutoff. If we go lower almost every op would end up compressed and we would gain minimal benefit from compression.